apps/admin_app/lib/admin_app_web/views/general_settings_view.ex

```diff
@@ -2,7 +2,13 @@ defmodule AdminAppWeb.GeneralSettingsView do
   use AdminAppWeb, :view
   alias Snitch.Data.Model.ProductBrand
 
+  @currencies ["USD", "INR", "GDP", "EUR"]
```

Comment:
This should go in the core rather than in the admin application IMO. @pkrawat1 @SagarKarwande ?

Reply:
Yes, valid currencies should be a part of core

Reply:
May I know the idea behind this?

Reply:
Any application would always depend on core and can easily access the valid currencies.